vuepress/docs/v6.3.2/tutorials/samples/hub.md

```diff
@@ -117,9 +117,9 @@ The Hub includes the following key components:
 - Entando Hub Login: This is an optional login component which can be used in a page’s top navigation.
 
 ### Micro Services
-A single Spring Boot microservice provides two REST endpoints. 
-- The first is a backend-for-a-frontend (BFF) service for the Hub UI and contains the various entity APIs
-- The second provides methods supporting the Entando App Builder integration (7.0.0+)
+A single Spring Boot microservice provides two REST endpoints:
+- The first is a backend-for-a-frontend (BFF) service for the Hub UI and contains the various entity APIs.
+- The second provides methods supporting the Entando App Builder integration (7.0.0+).
```

Comment:
The second provides methods "for" supporting the Entando App Builder integration (7.0.0+). ?

Reply:
...which support? that support... i need to better understand when to use "which" and when to use "that"... can we research and discuss in the next style guide meeting? @jyunmitch

Reply:
sure we can discuss at the meeting.

What I meant was to add the word "for" but wasn't sure. But 'that supports the Entando App ..." works too.